examples/sample-layers/s2-layer/s2-layer.js

```diff
@@ -64,13 +64,17 @@ const defaultProps = {
   getFillColor: f => f.fillColor || defaultFillColor
 };
 
-export default class S2Layer extends Layer {
+export default class S2Layer extends CompositeLayer {
+  constructor(props) {
+    super(props);
+  }
+
   initializeState() {
```

Comment:
I was referring to  `initializeState` but I guess the constructor could also be cut.